quizMySQL/QuizGame.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Android.App;
+using Android.Content;
+using Android.OS;
+using Android.Runtime;
+using Android.Views;
+using Android.Widget;
+using quizMySQL.DataBase;
+
+namespace quizMySQL
+{
+    class QuizGame
+    {
+        List<QuizQuestion> questionList;
+        public DbManager database { get; set; }
+        public int amountOfQuestions { get; set; }
+        public int currentQuestion { get; set; }
+        public int points { get; set; }
+
+
+        public QuizGame()
+        {
+            questionList = new List<QuizQuestion>();
+            amountOfQuestions = 8;
+            currentQuestion = 0;
+        }
+        public void populatingQuestionList(DbManager database)
+        {
+            //Making sure the list is empty
+            questionList.Clear();
+            for (int i = 0; i < amountOfQuestions; i++)
+            {
+                //picking  random question from database
+                var qq = new QuizQuestion(database.findRndQuestion().ExecuteReader());
+                //checking if the same question exists in question list
+                bool match = questionList.Any(x => x.ID == qq.ID);
+                //if exists pick another
+                while (match)
+                {
+                    qq = new QuizQuestion(database.findRndQuestion().ExecuteReader());
```

Comment:
it would be better to fetch all questions at once in the beginning and choose one at random from that list, instead of going to the db all the time [since questions don't change]

in other words the db should only be used to store data, it shouldn't have functionality like picking a random question